wire-ios/Wire-iOS Tests/AudioButtonOverlayTests.swift

```diff
@@ -16,6 +16,8 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 // 
 
+import XCTest
+
```

Comment:
We can get rid of that line.

Reply:
No we can't, since it's needed for `XCTAssertEqual`.

Reply:
I meant line 20.

Reply:
Sorry, done.